ssl-config-generator/index.html

```diff
@@ -334,7 +334,11 @@
                     }
                     if (data.hstsEnabled == "true") {
                         data.hsts = '\n    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)' + '\n' +
-                            '    add_header Strict-Transport-Security max-age=15768000;';
+                            '    add_header Strict-Transport-Security "max-age=15768000"';
```

Comment:
The final semicolon is added in line 341 below, but we have to do some logic before we're ready to terminate the directive (specifically, check if the server version if newer than 1.7.5).

The addition of the double quote is, I think, prophylactic here as `max-age=15768000` doesn't include any characters meaningful to the nginx parser. I'm not 100% sure since I haven't tried this config without the quotes, and all the examples I've seen delineate the header string this way. It's been a while since I stared deeply into the nginx config grammar, but I'm happy to chase that one down if you feel it's important.